File: solutions/beecrowd/2437/2437.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int x_m, y_m, x_r, y_r;

        while (input.hasNextInt()) {
            x_m = input.nextInt();
            y_m = input.nextInt();
            x_r = input.nextInt();
            y_r = input.nextInt();

            System.out.println(Math.abs(x_r - x_m) + Math.abs(y_r - y_m));
        }
    }
}

File: solutions/beecrowd/2437/2437.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    while (sc.hasNextInt()) {
        val x_m = sc.next().toInt();
        val y_m = sc.next().toInt();
        val x_r = sc.next().toInt();
        val y_r = sc.next().toInt();

        println("%d".format(Math.abs(x_r - x_m) + Math.abs(y_r - y_m)))
    }
}
